kyuubi-ctl/src/test/scala/org/apache/kyuubi/ctl/KyuubiCtlArgumentsSuite.scala

```diff
@@ -49,7 +49,7 @@ class KyuubiCtlArgumentsSuite extends KyuubiFunSuite {
       Seq("server", "engine").foreach { service =>
         val args = Seq(
           command, service,
-          "--zkAddress", zkAddress,
+          "--zk-quorum", zkAddress,
```

Comment:
should we also change the name of `zkAddress` and `ZK_ADDRESS`?

Reply:
thanks